contentcuration/contentcuration/assets/channelList/views/ChannelListIndex.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+  <VApp>
+    <AppBar>
+      <template #tabs>
+        <VTab
+          v-for="listType in lists"
+          :key="listType.id"
+          :to="getChannelLink(listType)"
+        >
+          <span v-if="listType === 'STARRED'"></span>
+          {{ $tr(listType) }}
+        </VTab>
+        <VTab
+          :to="channelSetLink"
+        >
+          {{ $tr("channelSets") }}
+        </VTab>
+      </template>
+    </AppBar>
+    <VContent>
+      <VContainer fluid>
+        <VCard v-if="invitations.length">
+          <VList subheader>
+            <VSubheader>{{ $tr('invitations') }}</VSubheader>
+            <VListTile
```

Comment:
Not really a huge thing, but would it make sense to follow the pattern with channel list and channel set list and make the invitation items separate components?

Reply:
Possibly - I kept it within the overall ChannelListIndex because it was relatively small, and without it, there wasn't much going on in the ChannelListIndex, so this seemed like a way to have a more substantial component, and not create additional single use components.